repose-aggregator/tests/release-verification/src/fake-services/fake-keystone2/package.json

```diff
@@ -1,22 +1,22 @@
 {
-  "name": "ReposeFakeKeystonev2",
-  "version": "0.1.0",
-  "description": "Mock Keystone Service",
-  "readme": "README.md",
+  "name": "service-mocks",
+  "version": "1.0.0",
```

Comment:
Honestly, I don't think we'll be publishing this, so it really doesn't make much difference.

I like to start initial releases at a major, which is how I treated this module.

I'll drop it to `0.2.0` if you want, though.